### DISCUSSION TITLE
Updating data in the isolate

### DISCUSSION THREAD

**Discussion by andreystavitsky** [flutterdata/flutter_data#227] [Q&A] [answered]
For example, I have an FCM onBackgroundMessage function that updates data in the repository, and I have a watcher (watchAll) on my app screen. Is it possible to update data in the background and reflect that update on the screen? What is the correct setup for that situation?

**Comment by frank06** [comment, marked as answer]
Sadly it's not possible to access FD in an isolate because at the moment it's backed by Hive and this is a Hive limitation. Should be possible the day we support other storage backends. In the meantime maybe passing data through shared preferences or similar could work
